plugins/aws/resoto_plugin_aws/resource/cloudfront.py

```diff
@@ -0,0 +1,1497 @@
+    cached_methods: Optional[AwsCloudFrontCachedMethods] = field(default=None)
+
+
+@define(eq=False, slots=False)
+class AwsCloudFrontLambdaFunctionAssociation:
+    kind: ClassVar[str] = "aws_cloudfront_lambda_function_association"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "lambda_function_arn": S("LambdaFunctionARN"),
+        "event_type": S("EventType"),
+        "include_body": S("IncludeBody"),
+    }
+    lambda_function_arn: Optional[str] = field(default=None)
+    event_type: Optional[str] = field(default=None)
+    include_body: Optional[bool] = field(default=None)
+
+
+@define(eq=False, slots=False)
+class AwsCloudFrontLambdaFunctionAssociations:
+    kind: ClassVar[str] = "aws_cloudfront_lambda_function_associations"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "quantity": S("Quantity"),
+        "items": S("Items", default=[]) >> ForallBend(AwsCloudFrontLambdaFunctionAssociation.mapping),
+    }
+    quantity: Optional[int] = field(default=None)
+    items: List[AwsCloudFrontLambdaFunctionAssociation] = field(factory=list)
+
+
+@define(eq=False, slots=False)
+class AwsCloudFrontFunctionAssociation:
+    kind: ClassVar[str] = "aws_cloudfront_function_association"
+    mapping: ClassVar[Dict[str, Bender]] = {"function_arn": S("FunctionARN"), "event_type": S("EventType")}
+    function_arn: Optional[str] = field(default=None)
+    event_type: Optional[str] = field(default=None)
+
+
+@define(eq=False, slots=False)
+class AwsCloudFrontFunctionAssociations:
+    kind: ClassVar[str] = "aws_cloudfront_function_associations"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "quantity": S("Quantity"),
+        "items": S("Items", default=[]) >> ForallBend(AwsCloudFrontFunctionAssociation.mapping),
+    }
+    quantity: Optional[int] = field(default=None)
+    items: List[AwsCloudFrontFunctionAssociation] = field(factory=list)
+
+
+@define(eq=False, slots=False)
+class AwsCloudFrontCookieNames:
+    kind: ClassVar[str] = "aws_cloudfront_cookie_names"
+    mapping: ClassVar[Dict[str, Bender]] = {"quantity": S("Quantity"), "items": S("Items", default=[])}
+    quantity: Optional[int] = field(default=None)
+    items: List[str] = field(factory=list)
+
+
+@define(eq=False, slots=False)
+class AwsCloudFrontCookiePreference:
+    kind: ClassVar[str] = "aws_cloudfront_cookie_preference"
+    mapping: ClassVar[Dict[str, Bender]] = {
+        "forward": S("Forward"),
+        "whitelisted_names": S("WhitelistedNames") >> Bend(AwsCloudFrontCookieNames.mapping),
+    }
+    forward: Optional[str] = field(default=None)
+    whitelisted_names: Optional[AwsCloudFrontCookieNames] = field(default=None)
+
+
+@define(eq=False, slots=False)
+class AwsCloudFrontHeaders:
+    kind: ClassVar[str] = "aws_cloudfront_headers"
+    mapping: ClassVar[Dict[str, Bender]] = {"quantity": S("Quantity"), "items": S("Items", default=[])}
+    quantity: Optional[int] = field(default=None)
+    items: List[str] = field(factory=list)
```

Comment:
Please go through the list and remove all other occurrences of List abstractions.